app/models/manageiq/providers/hawkular/middleware_manager/event_catcher_mixin.rb

```diff
@@ -0,0 +1,67 @@
+# We use this to provide easy access to both the runner and specs
+module ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcherMixin
+  # extend ActiveSupport::Concern
+
+  # TODO: perhaps we need to block certain events.  Another way to do this would be to add a tag on events we want to
+  # suck into MiQ, and just fetch with the tag as criteria
+  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze # some Hash of included events
+  EXCLUDE_EVENTS = {}.freeze # TODO: some Hash of exclusions for the included events
```

Comment:
This inclusion and exclusion is actually handled in a different manner.  I'll try to track down the details and get back to this.